server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMailboxes.feature

```diff
@@ -286,3 +286,12 @@ Feature: GetMailboxes method
     Given "bob@domain.tld" deletes the mailbox "shared" owned by "alice@domain.tld"
     When "alice@domain.tld" lists mailboxes
     Then the mailboxes should contain "shared" in "Personal" namespace
+
+  Scenario: A sharee should not be able to create a shared mailbox child
+    Given "bob@domain.tld" creates mailbox "sharedChild" with creationId "c-01" in mailbox "shared" owned by "alice@domain.tld"
```

Comment:
This line is same as the line 296 but the results are not the same, it make me confuse

Reply:
Ok, one is an assert on the response of this request
one other is an assert on the side effect of the request

I chose not to mix them, because I found it way more clear ;-)

(And we already had the pattern of splitting them in conventional integration tests).